orchestra/management/commands/migrate_certifications.py

```diff
@@ -0,0 +1,26 @@
+from django.core.management.base import BaseCommand
+from orchestra.workflow.certifications import migrate_certifications
+
+
+class Command(BaseCommand):
+    help = 'Migrates specified certifications across workflows.'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'workflow_src',
```

Comment:
nit: spell it out! (this one is easy because no one ever has to type it in at all).